### PR TITLE
Added Krush Solder (60% & 65%)

### PR DESCRIPTION
**Pull request by MaiTheSan**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/sawnsprojects/krush/krush60/solder/config.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#define RGB_DI_PIN F1
+#ifdef RGB_DI_PIN
+    #define RGBLED_NUM 20
+    #define RGBLIGHT_HUE_STEP 8
+    #define RGBLIGHT_SAT_STEP 8
+    #define RGBLIGHT_VAL_STEP 8
+    #define RGBLIGHT_LIMIT_VAL 200 /* The maximum brightness level */
+    #define RGBLIGHT_SLEEP  /* If defined, the RGB lighting will be switched off when the host goes to sleep */
+/*== all animations enable ==*/
+    #define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/sawnsprojects/krush/krush60/solder/config.h:50]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/sawnsprojects/krush/krush60/solder/rules.mk [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+CONSOLE_ENABLE  = yes       # Console for debug
+COMMAND_ENABLE  = no        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+
+LAYOUTS = 60_ansi 60_ansi_arrow 60_ansi_arrow_split_bs_7u_spc
```
Comment (drashna) [inline keyboards/sawnsprojects/krush/krush60/solder/rules.mk:22]:
```suggestion
LAYOUTS = 60_ansi 60_ansi_arrow 60_ansi_arrow_split_bs_7u_spc

```

File: keyboards/sawnsprojects/krush/krush65/solder/config.h [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+#define RGB_DI_PIN F1
+#ifdef RGB_DI_PIN
+    #define RGBLED_NUM 20
+    #define RGBLIGHT_HUE_STEP 8
+    #define RGBLIGHT_SAT_STEP 8
+    #define RGBLIGHT_VAL_STEP 8
+    #define RGBLIGHT_LIMIT_VAL 200 /* The maximum brightness level */
+    #define RGBLIGHT_SLEEP  /* If defined, the RGB lighting will be switched off when the host goes to sleep */
+/*== all animations enable ==*/
+    #define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/sawnsprojects/krush/krush65/solder/config.h:58]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

**Review by drashna — CHANGES_REQUESTED** [review]
You have some issuse with the git submodules that neds to ber resolved

**Comment by MaiTheSan** [comment]
resolved. Thanks Drashna. ehe

**Comment by tzarc** [comment]
1. Submodules are still wrong, despite you saying you've resolved them.
2. Keyboards that aren't part of this PR are present in your PR.
3. You have leftover files that should exist: `keyboards/sawnsprojects/krush/krush60/solder/keymaps/60_ansi_splt_bs - Copy`
4. Most importantly, you've ticked the "PR Checklist" item in the description as "complete", yet the majority of the issues in this PR are directly listed.

Please go back and re-read and apply the changes in the PR checklist. It's there for you to make sure you're not wasting maintainers' time -- until you've done so, any further reviews on this PR by collaborators will end up pointing out issues that you can discover and solve on your own.

**Pull request closed without merging**